src/components/Select/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { InputWrapper, SelectFieldSet } from './Select.styled';
+
+const Select = ({ label, id, options }) => {
+    const [value, setValue] = useState('');
+
+    const handleChange = (e) => {
```

Comment:
For such simple functions you can remove brackets: const handleChange = (e) => setValue(e.target.value);

Reply:
Removed